### PR TITLE
samples: updates samples to v1 (1 of 8)

### PR DESCRIPTION
**Pull request by telpirion**
This updates the java-aiplatform samples to use the v1 version of the samples.

### CODE REVIEW

**Comment by telpirion** [comment]
Note: I'm still working on this PR a little bit (adding in resource IDs for tests)--I just wanted to get more results from CI. I'll repost when it's fully ready for review.

**Comment by telpirion** [comment]
Samples now ready for review. This PR also picked up some linter fixes unrelated to the primary changes.

File: samples/snippets/src/main/java/aiplatform/CreateBatchPredictionJobTextClassificationSample.java [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+              .build();
+      BatchPredictionJob batchPredictionJob =
+          BatchPredictionJob.newBuilder()
+              .setDisplayName(displayName)
+              .setModel(modelName)
+              .setInputConfig(inputConfig)
+              .setOutputConfig(outputConfig)
+              .build();
+      LocationName parent = LocationName.of(project, location);
+      BatchPredictionJob response = client.createBatchPredictionJob(parent, batchPredictionJob);
```
Comment (stephaniewang526) [inline samples/snippets/src/main/java/aiplatform/CreateBatchPredictionJobTextClassificationSample.java:83]: is there an `ApiException` we can catch here if this operation didn't go through?

File: samples/snippets/src/main/java/aiplatform/CreateBatchPredictionJobTextEntityExtractionSample.java [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+              .build();
+      BatchPredictionJob batchPredictionJob =
+          BatchPredictionJob.newBuilder()
+              .setDisplayName(displayName)
+              .setModel(modelName)
+              .setInputConfig(inputConfig)
+              .setOutputConfig(outputConfig)
+              .build();
+      LocationName parent = LocationName.of(project, location);
+      BatchPredictionJob response = client.createBatchPredictionJob(parent, batchPredictionJob);
```
Comment (stephaniewang526) [inline samples/snippets/src/main/java/aiplatform/CreateBatchPredictionJobTextEntityExtractionSample.java:83]: Same as above -- is there an `ApiException` we can catch here if this operation didn't go through?

File: samples/snippets/src/main/java/aiplatform/CreateBatchPredictionJobTextSentimentAnalysisSample.java [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+              .build();
+      BatchPredictionJob batchPredictionJob =
+          BatchPredictionJob.newBuilder()
+              .setDisplayName(displayName)
+              .setModel(modelName)
+              .setInputConfig(inputConfig)
+              .setOutputConfig(outputConfig)
+              .build();
+      LocationName parent = LocationName.of(project, location);
+      BatchPredictionJob response = client.createBatchPredictionJob(parent, batchPredictionJob);
```
Comment (stephaniewang526) [inline samples/snippets/src/main/java/aiplatform/CreateBatchPredictionJobTextSentimentAnalysisSample.java:83]: Same as above -- is there an `ApiException` we can catch here if this operation didn't go through?

File: samples/snippets/src/test/java/aiplatform/CreateBatchPredictionJobTextClassificationSampleTest.java [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+            batchPredictionDisplayName,
+            MODEL_ID,
+            GCS_SOURCE_URI,
+            GCS_OUTPUT_URI);
+
+    // Assert
+    String got = bout.toString();
+    assertThat(got).contains(batchPredictionDisplayName);
+    assertThat(got).contains("response:");
+    batchPredictionJobId = got.split("name:")[1].split("batchPredictionJobs/")[1].split("\"\n")[0];
```
Comment (stephaniewang526) [inline samples/snippets/src/test/java/aiplatform/CreateBatchPredictionJobTextClassificationSampleTest.java:107]: Not sure if this is necessary? Is this meant to be printed out or checked?

File: samples/snippets/src/test/java/aiplatform/CreateBatchPredictionJobTextClassificationSampleTest.java [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+    // Act
+    String batchPredictionDisplayName =
+        String.format(
+            "temp_java_create_batch_prediction_TCN_%s",
+            UUID.randomUUID().toString().replaceAll("-", "_").substring(0, 26));
+
+    CreateBatchPredictionJobTextClassificationSample
+        .createBatchPredictionJobTextClassificationSample(
+            PROJECT,
+            "us-central1",
```
Comment (stephaniewang526) [inline samples/snippets/src/test/java/aiplatform/CreateBatchPredictionJobTextClassificationSampleTest.java:97]: To be consistent with the rest, maybe make this a `private static final` variable as well?

File: samples/snippets/src/test/java/aiplatform/CreateBatchPredictionJobTextEntityExtractionSampleTest.java [hunk 6]
```diff
@@ -0,0 +1,109 @@
...
+    // Act
+    String batchPredictionDisplayName =
+        String.format(
+            "temp_java_create_batch_prediction_TEN_%s",
+            UUID.randomUUID().toString().replaceAll("-", "_").substring(0, 26));
+
+    CreateBatchPredictionJobTextEntityExtractionSample
+        .createBatchPredictionJobTextEntityExtractionSample(
+            PROJECT,
+            "us-central1",
```
Comment (stephaniewang526) [inline samples/snippets/src/test/java/aiplatform/CreateBatchPredictionJobTextEntityExtractionSampleTest.java:97]: Same as above -- To be consistent with the rest, maybe make this a `private static final` variable as well?

File: samples/snippets/src/test/java/aiplatform/CreateBatchPredictionJobTextEntityExtractionSampleTest.java [hunk 7]
```diff
@@ -0,0 +1,109 @@
...
+            batchPredictionDisplayName,
+            MODEL_ID,
+            GCS_SOURCE_URI,
+            GCS_OUTPUT_URI);
+
+    // Assert
+    String got = bout.toString();
+    assertThat(got).contains(batchPredictionDisplayName);
+    assertThat(got).contains("response:");
+    batchPredictionJobId = got.split("name:")[1].split("batchPredictionJobs/")[1].split("\"\n")[0];
```
Comment (stephaniewang526) [inline samples/snippets/src/test/java/aiplatform/CreateBatchPredictionJobTextEntityExtractionSampleTest.java:107]: Same as above -- Not sure if this is necessary? Is this meant to be printed out or checked?

File: samples/snippets/src/test/java/aiplatform/CreateBatchPredictionJobTextSentimentAnalysisSampleTest.java [hunk 8]
```diff
@@ -0,0 +1,109 @@
...
+    // Act
+    String batchPredictionDisplayName =
+        String.format(
+            "temp_java_create_batch_prediction_TSN_%s",
+            UUID.randomUUID().toString().replaceAll("-", "_").substring(0, 26));
+
+    CreateBatchPredictionJobTextSentimentAnalysisSample
+        .createBatchPredictionJobTextSentimentAnalysisSample(
+            PROJECT,
+            "us-central1",
```
Comment (stephaniewang526) [inline samples/snippets/src/test/java/aiplatform/CreateBatchPredictionJobTextSentimentAnalysisSampleTest.java:97]: Same as above - ptal.

File: samples/snippets/src/test/java/aiplatform/CreateBatchPredictionJobTextSentimentAnalysisSampleTest.java [hunk 9]
```diff
@@ -0,0 +1,109 @@
...
+            batchPredictionDisplayName,
+            MODEL_ID,
+            GCS_SOURCE_URI,
+            GCS_OUTPUT_URI);
+
+    // Assert
+    String got = bout.toString();
+    assertThat(got).contains(batchPredictionDisplayName);
+    assertThat(got).contains("response:");
+    batchPredictionJobId = got.split("name:")[1].split("batchPredictionJobs/")[1].split("\"\n")[0];
```
Comment (stephaniewang526) [inline samples/snippets/src/test/java/aiplatform/CreateBatchPredictionJobTextSentimentAnalysisSampleTest.java:107]: Same as above - ptal.

File: samples/snippets/src/main/java/aiplatform/CreateBatchPredictionJobTextClassificationSample.java (hunk 1 above)
Comment (telpirion) [reply samples/snippets/src/main/java/aiplatform/CreateBatchPredictionJobTextClassificationSample.java:83]: Yes there is! Should I put in a explicit `try...catch` or is adding `throws ApiException` to the method signature sufficient?

Comment (telpirion) [reply samples/snippets/src/main/java/aiplatform/CreateBatchPredictionJobTextClassificationSample.java]: Done.

File: samples/snippets/src/main/java/aiplatform/CreateBatchPredictionJobTextEntityExtractionSample.java (hunk 2 above)
Comment (telpirion) [reply samples/snippets/src/main/java/aiplatform/CreateBatchPredictionJobTextEntityExtractionSample.java]: Done.

File: samples/snippets/src/main/java/aiplatform/CreateBatchPredictionJobTextSentimentAnalysisSample.java (hunk 3 above)
Comment (telpirion) [reply samples/snippets/src/main/java/aiplatform/CreateBatchPredictionJobTextSentimentAnalysisSample.java]: Done.

File: samples/snippets/src/test/java/aiplatform/CreateBatchPredictionJobTextClassificationSampleTest.java (hunk 4 above)
Comment (telpirion) [reply samples/snippets/src/test/java/aiplatform/CreateBatchPredictionJobTextClassificationSampleTest.java]: I moved this to the `tearDown()`.

File: samples/snippets/src/test/java/aiplatform/CreateBatchPredictionJobTextClassificationSampleTest.java (hunk 5 above)
Comment (telpirion) [reply samples/snippets/src/test/java/aiplatform/CreateBatchPredictionJobTextClassificationSampleTest.java]: Done.

File: samples/snippets/src/test/java/aiplatform/CreateBatchPredictionJobTextEntityExtractionSampleTest.java (hunk 6 above)
Comment (telpirion) [reply samples/snippets/src/test/java/aiplatform/CreateBatchPredictionJobTextEntityExtractionSampleTest.java]: Done.

File: samples/snippets/src/test/java/aiplatform/CreateBatchPredictionJobTextEntityExtractionSampleTest.java (hunk 7 above)
Comment (telpirion) [reply samples/snippets/src/test/java/aiplatform/CreateBatchPredictionJobTextEntityExtractionSampleTest.java]: I moved it to the `tearDown()`.

File: samples/snippets/src/test/java/aiplatform/CreateBatchPredictionJobTextSentimentAnalysisSampleTest.java (hunk 8 above)
Comment (telpirion) [reply samples/snippets/src/test/java/aiplatform/CreateBatchPredictionJobTextSentimentAnalysisSampleTest.java]: Done.

**Review by stephaniewang526 — APPROVED** [review]

**Pull request merged**